internal/pkg/template/workload.go

```diff
@@ -234,6 +242,11 @@ func (tg HTTPTargetContainer) IsHTTPS() bool {
 	return tg.Port == "443"
 }
 
+// IsEqual returns true if httpContainerPort and port mapping port are equal.
+func IsEqual(s string, p uint16) bool {
```

Comment:
The function name is a bit too generic for the given parameter types 💭 . For example, a client might think to use `isEqual struct1{} struct2{}`.

What do you think of instead adding the following function:
```go
func strconvUint16(val uint16) string { return strconv.FormatUint(uint64(val), 10) } 
```
and in the Go template we can do `{{eq port1 (strconvUint16 port2)}}`